src/lister_haxe/lister.hx

```diff
@@ -0,0 +1,38 @@
+import haxe.io.Path;
+using StringTools;
+
+class Lister {
+  static public function get(url:String):Array<String> {
+    var http = new haxe.Http(url);
+    var hits = [];
+
+    http.onData = function(resp) {
```

Comment:
["lvalue" (declared) vs "rvalue" (assigned) functions](http://haxe.org/manual/expression-function.html) - Haxe doesn't take types on functions defined as a function expression.  Which is a little weird.  I probably could define these (`onData` and `onError`) as class functions instead.